### ISSUE TITLE
Support Ternary Operator 

### ISSUE THREAD

**Issue by st0012** [goby-lang/goby#677]
For example: `foo ? bar : baz`

**Comment by hachi8833** [comment]
I thought you wouldn't adopt ternary operators in Goby 😲

**Comment by st0012** [comment]
Yep I didn’t want to support this, but this is kind of a basic component to programming language I think

**Comment by eliothedeman** [comment]
Is this something people have complained about not having?

**Comment by st0012** [comment]
Not yet, just I think it would be. But it might also because we don't really have much users for now.

**Comment by Maxwell-Alexius** [comment]
Just a thought, do you want to do something like in Python style (although Goby is referenced to Ruby style syntax):
```
hello = 'world' if created? else 'nothing'
```

But this style is kind of like inline conditional which is addressed in #580

**Comment by st0012** [comment]
@Maxwell-Alexius I prefer using Ruby style because python style might confuse users

**Comment by spavikevik** [comment]
Would you guys let me try to attack this in my free time? I love Ruby and Go, and have tried working with compilers/interpreters before, so I feel like it wouldn't be such an idea to attempt something like this.

**Comment by st0012** [comment]
@spavikevik sure, feel free to work on it 😄
